src/server/utils/SettingsStorage.ts

```diff
@@ -18,17 +19,23 @@ export const loadSettings = () => {
 }
 
 export const saveSettings = () => {
-    var json = JSON.stringify(reduxState.settings[0].generics)
+    let stringifiedSettings = JSON.stringify(reduxState.settings[0].generics)
     if (!fs.existsSync('storage')) {
         fs.mkdirSync('storage')
     }
-
-    fs.writeFile(
-        path.resolve('storage', 'settings.json'),
-        json,
-        'utf8',
-        (error) => {
-            console.log(error)
-        }
-    )
+    if (
+        stringifiedSettings !==
+        JSON.stringify(defaultSettingsReducerState()[0].generics)
+    ) {
+        fs.writeFile(
+            path.resolve('storage', 'settings.json'),
+            stringifiedSettings,
+            'utf8',
+            (error) => {
+                console.log(error)
+            }

```

Comment:
Can be simplified to this:
```suggestion
            (error) => console.log(error),

```
And perhaps warn or error would be better suited log level.

Reply:
I didn't touch logging, because next plan is to update to mediatech logger.
There's still a lot of console.log() in the code.